app/views/api/_observation.xml.builder

```diff
@@ -30,6 +30,18 @@ xml.tag!(tag,
         xml_detailed_object(xml, :sequence, sequence)
       end
     end
+    xml.notes do
+      object.notes.each do |key, value|
+        xml.notes_part do
+          xml_string(xml, :key, key)
+          xml_html_string(xml, :value, value.tl)
+        end
+      end
+    end
+    xml_datetime(xml, :created_at, object.created_at)
+    xml_datetime(xml, :updated_at, object.updated_at)
+    xml_integer(xml, :number_of_views, object.num_views)
+    xml_datetime(xml, :last_viewed, object.last_view)
```

Comment:
I don't understand the comment on line 51: "# Do it this way, else will not use eager-loaded image instance."

Also note the difference from  _observation.json.builder analagous lines (copied below), which:
- when finding the thumbnail, build a list of non-thumbnails (`other_images`), and
- then use that list, versus again iterating over all the images.  
```ruby
  object.images.each do |image|
    # Do it this way, else will not use eager-loaded image instance.
    if image.id == object.thumb_image_id
      json.primary_image { json_detailed_object(json, image) }
    else
      other_images << image
    end
  end
  json.images other_images.map do |image|
    json_detailed_object(json, image)
  end
```
Comment:
- The two builders really should work the same way.
- Is it worth creating the new list? The difference between `object.images` and `other_images` is just one image.